examples/asa/scripts/transfer/gold-contract-sc.js

```diff
@@ -22,6 +21,22 @@ async function run(runtimeEnv, deployer) {
 
   // Transaction FAIL as Elon tried to receive instead of John
   await transferAlgo(deployer, { addr: sender}, elonMuskAccount.addr, 50, lsig);
+
+  //group transactions
+  let transactions = [
+    mkTxnParams({addr: sender}, johnAccount.addr, 30, lsig, {}),
+    mkTxnParams({addr: sender}, johnAccount.addr, 40, lsig, {}),
+    mkTxnParams({addr: sender}, johnAccount.addr, 50, lsig, {})]
+  
+  // Transaction PASS - for all transactions amount <= 100 and receiver is john
+  await transferAlgoAtomic(deployer, transactions);
+  
+  transactions = [
+    mkTxnParams({addr: sender}, johnAccount.addr, 200, lsig, { note: "ALGO PAID" }),
+    mkTxnParams({addr: sender}, johnAccount.addr, 20, lsig, { note: "ALGO PAID" })]
+  
+  // Transaction FAIL - since for the first transaction amount = 200 (which is > 100, so both transactions will fail)
+  await transferAlgoAtomic(deployer, transactions);
 }
```

Comment:
I don't think we should merge group transaction example in our current examples. It may be confusing to the user.
also we have different ref-template for group transactions only. what do you think ?

Reply:
yes, these will be moved to ref-templates only. Just logged it here if someone wanted to test before rolling out the next PR (as functions will be identical).

Reply:
OK. If you want to move it in other PR, then please add a task (or a subtask) in pivotal tracker.